tests/ssl/client-certs/server/hello.proto

```
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package hello;
```

Buf (1.71.0)

[error] Files with package "hello" must be within a directory "hello" relative to root but were in directory "tests/ssl/client-certs/server".

(PACKAGE_DIRECTORY_MATCH)